src/main/java/Duke.java

```diff
@@ -1,20 +1,51 @@
-import java.util.Scanner;
 import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
-    private static String[] Tasks = new String[100];
+    private static String[] tasks = new String[100];
```

Comment:
I feel that you can use a constant variable to replace "100" as it is a magic number